tools/clang/include/clang/AST/HlslTypes.h

```diff
@@ -80,6 +82,12 @@ enum HLSLScalarType {
   HLSLScalarType_uint8_4packed
 };
 
+enum HLSLMatrixOrientation {
```

Comment:
Why don't we re-use the `hlsl::MatrixOrientation` enum out of `DxilTypeSystem.h` instead of creating a new enum?

Reply:
Orientation in HLSLMatrixOrientation is to match the is_row_major in matrix.internal::matrix.
Column is 0, Row is 1.
Default is only used to save shorthanded type and set to 2.

Reply:
Why don't we re-use the hlsl::MatrixOrientation enum out of DxilTypeSystem.h instead of creating a new enum?

(you didn't actually explain why you need a new enum that has the same case values)